src/styles/sass-styles/sass-styles.scss

```diff
@@ -0,0 +1,55 @@
+.container {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+  font-size: 30px;
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+.calendar {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  width: 7 * $cell-size + 10px;
+  gap: 1px;
+
+  &__day {
+    @for $index from 1 through 31 {
+      &:nth-child(#{$index})::before {
+        content: "#{$index}";
+        display: flex;
+        box-sizing: border-box;
+        align-items: center;
+        justify-content: center;
+        border: 1px solid black;
+        width: $cell-size;
+        height: $cell-size;
+        background-color: $cell-color;
+      }
+
+      &:hover::before {
+        cursor: pointer;
+        background-color: $cell-hover;
+        transition: 0.5s;
```

Comment:
```suggestion
        background-color: $cell-color;
        transition: 0.5s;
      }

      &:hover::before {
        cursor: pointer;
        background-color: $cell-hover;
```
for make smooth changes in two ways transition should be settled for element selector instead hover selector